### PR TITLE
updates mlctl init (#18)

### PR DESCRIPTION
**Pull request by deepk2u**
## Goals of this PR :tada:
https://github.com/intuit/mlctl/issues/18

### CODE REVIEW

File: mlctl/clis/template/{{cookiecutter.package_name}}/model/train.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+def train():
+    print('Hello, put your model training code here')
+
+
+def main(ta):
+    """
+    Load features and labels, train the tree, and serialize model artifact.
+    Note: This is the training entrypoint used by baklava!
+    """
```
Comment (awcchungster) [inline mlctl/clis/template/{{cookiecutter.package_name}}/model/train.py:14]: I think there should be a `train()` call within main

Comment (awcchungster) [reply mlctl/clis/template/{{cookiecutter.package_name}}/model/train.py:14]: correction, based on the setup.py, the `train()` is called. In that case, the training adapter should be embedded there

File: tests/clis/mlctl_init/setup.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    install_requires=[
+            'mlsriracha>=0.0.2'
+        ],
+    python_requires='>=3.6',
+    entry_points={
+        'mlbaklava.train': [
+            'example_train = model.train:train'
+        ],
+        'mlbaklava.predict': [
+            'example_predict = model.predict:predict',
```
Comment (awcchungster) [inline tests/clis/mlctl_init/setup.py:17]: For hosting, I think we are missing the load model on container start. Refer to the examples here:

https://github.com/intuit/baklava/tree/master/examples/7-predict-initializer

**Review by awcchungster — CHANGES_REQUESTED** [review]
Minor changes to the init structure and functions

**Comment by srivathsanvc** [comment]
@deepk2u - can you please update based on the review comments?

File: mlctl/clis/template/cookiecutter.json [hunk 3]
```diff
@@ -1,6 +1,4 @@
 {
-  "project_name": "Boilerplate ML Model",
-  "project_slug": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}",
-  "project_short_description": "Python Boilerplate ML model contains all the boilerplate you need to create a working ML model project.",
-  "version": "0.1.0"
+  "package_name": "Boilerplate ML Model",
```
Comment (srivathsanvc) [inline mlctl/clis/template/cookiecutter.json:2]:
```suggestion
  "package_name": "Sample ML Model",
```

File: mlctl/clis/template/cookiecutter.json [hunk 4]
```diff
@@ -1,6 +1,4 @@
...
+  "description": "Python Boilerplate ML model contains all the boilerplate you need to create a working ML model project."
```
Comment (srivathsanvc) [inline mlctl/clis/template/cookiecutter.json:3]:
```suggestion
  "description": "Sample ML model contains all the boilerplate code required to create a working ML model project."
```

File: mlctl/clis/template/{{cookiecutter.package_name}}/model/train.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import uuid
+import os
+import shutil
+
+
+def train():
```
Comment (srivathsanvc) [inline mlctl/clis/template/{{cookiecutter.package_name}}/model/train.py:6]:
```suggestion
def train(ta):
```

**Pull request closed without merging**